common/data_refinery_common/models/models.py

```diff
@@ -324,6 +324,13 @@ def get_sample_metadata_fields(self):
                 fields.append(field)
         return fields
 
+    def get_sample_technologies(self):
+        """ Get all metadata fields that are non-empty for at least one sample in the experiment.
+        See https://github.com/AlexsLemonade/refinebio-frontend/issues/211 for why this is needed.
+        """
+        technologies = list(filter(None, list(set([t['technology'] for t in self.samples.all().values('technology')]))))
```

Comment:
The Jone's Complexity on this line seems a bit high. Also I think it might be a bit simpler to just append `if t['technology'] is not None` onto that list comprehension than it is to filter and cast back to a list.